city_metrix/layers/layer_tools.py

```diff
@@ -85,3 +68,90 @@ def get_haversine_distance(lon1, lat1, lon2, lat2):
 
     return distance
 
+def build_cache_layer_names(layer_obj):
```

Comment:
I think all the functions here down in this file could use some docstrings to explain what they're trying to do, because they're all fairly complicated and seem to be trying to something very specific that's not well-expressed by just the function name